akka-persistence/src/test/scala/akka/persistence/PersistentActorSpec.scala

```diff
@@ -248,6 +248,14 @@ object PersistentActorSpec {
         persistAsync(event) { evt ⇒ sender() ! s"${evt.data}-b-${sendMsgCounter.incrementAndGet()}" }
     }
   }
+  class PersistAllNilPersistentActor(name: String) extends ExamplePersistentActor(name) {
+
+    val receiveCommand: Receive = commonBehavior orElse {
+      case Cmd(data) ⇒
+        persistAll(Nil)(_ ⇒ ())
```

Comment:
Yeah, the callback is never called so doesn't really matter if we do not call it without looping vs. not call it with looping :)